templates/repo/actions/runs_list.tmpl

```diff
@@ -6,20 +6,24 @@
 	</div>
 	{{end}}
 	{{range .Runs}}
-		<li class="item gt-df gt-py-4">
+		<li class="item gt-df gt-py-3">
 			<div class="issue-item-left issue-item-icon gt-df gt-items-start">
 				{{template "repo/actions/status" (dict "status" .Status.String "locale" $.locale)}}
 			</div>
-			<div class="issue-item-main action-item-main gt-f1 gt-fc gt-df gt-mr-3">
+			<div class="issue-item-main action-item-main gt-f1 gt-fc gt-df gt-mr-3 gt-gap-2">
 				<div class="issue-item-top-row gt-mb-2">
 					<a class="index gt-no-underline title action-item-title" title="{{.Title}}" href="{{if .Link}}{{.Link}}{{else}}{{$.Link}}/{{.Index}}{{end}}">
 						{{- .Title -}}
 					</a>
 				</div>
 				<div class="desc issue-item-bottom-row gt-df gt-ac gt-fw">
-					<b>{{if not $.CurWorkflow}}{{.WorkflowID}} {{end}}#{{.Index}}</b>: {{$.locale.Tr "actions.runs.commit"}}
-					<a href="{{$.RepoLink}}/commit/{{.CommitSHA}}">{{ShortSha .CommitSHA}}</a>{{$.locale.Tr "actions.runs.pushed_by"}} {{.TriggerUser.GetDisplayName}}
-					<span class="ui label">
+					<b>{{if not $.CurWorkflow}}{{.WorkflowID}} {{end}}#{{.Index}}</b>
+					<span class="ui label gt-px-1 gt-mx-2">
+						<a class="gt-mono" href="{{$.RepoLink}}/commit/{{.CommitSHA}}">{{ShortSha .CommitSHA}}</a>
+					</span>
+						{{$.locale.Tr "actions.runs.pushed_by"}}
+						<a href="{{.TriggerUser.HomeLink}}">{{.TriggerUser.GetDisplayName}}</a>
+					<span class="ui label gt-px-1">
```

Comment:
Not aligned?

![image](https://github.com/go-gitea/gitea/assets/2114189/319cb9c9-4431-4718-b8bb-24b0c869066d)